src/applications/appeals/shared/tests/validatiions/areaOfDisagreements.unit.spec.js

```diff
@@ -51,7 +51,7 @@ describe.skip('areaOfDisagreementMaxLength', () => {
     const errors = { addError: sinon.spy() };
     areaOfDisagreementMaxLength(
       errors,
-      'ab '.repeat(MAX_LENGTH.ISSUE_NAME / 2),
+      'ab '.repeat(MAX_LENGTH.NOD_ISSUE_NAME / 2),
```

Comment:
Hmmm, now that I look at this, I think this should have been `DISAGREEMENT_REASON` not `ISSUE_NAME`. What do you think?